kotlin/spring/build.gradle.kts

```
@@ -0,0 +1,39 @@
+plugins {
+	id("org.springframework.boot") version "3.3.1"
+	id("io.spring.dependency-management") version "1.1.5"
+	kotlin("jvm") version "1.9.24"
+	kotlin("plugin.spring") version "1.9.24"
```

```suggestion
	id("org.springframework.boot") version "3.3.+"
	id("io.spring.dependency-management") version "+"
	kotlin("jvm") version "1.9.+"
	kotlin("plugin.spring") version "1.9.+"
```

to make sure we use the patched version of each